actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,31 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
+        delegate :==, :each, :empty?, to: :@variants
```

Comment:
I thought more tests failed without it, but it seems that only one does, and that's because the test itself uses `assert_equal` to compare `response.variant` with a symbol. I'm fine with removing it.